azure-pipelines.yml

```diff
@@ -90,3 +90,4 @@ jobs:
       /c/bazel/bazel.exe build "@stackage-pinning-test//:hspec"
 
     displayName: 'Run Bazel'
+    timeoutInMinutes: 120
```

Comment:
Is there an issue for this on the GHC tracker?

Reply:
It turns out that the initial run just happened to be a slow run. Subsequent runs were all of comparable length to previously. So it's unclear whether this is an upstream issue. But bumping up `timeoutInMinutes` should have the effect of making CI more reliable.